stdlib/contrib/bonitoo-io/servicenow/servicenow.flux

```diff
@@ -102,53 +118,65 @@ event = (
     return http.post(headers: headers, url: url, data: body)
 }
 
-// endpoint creates the endpoint for the ServiceNow external service.
+// endpoint sends events to [ServiceNow](https://servicenow.com/) using data from input rows.
 //
 // ## Parameters
-//
-// - `url` ServiceNow web service URL.
-// - `username` username for HTTP BASIC authentication. Default is empty string (no authentication).
-// - `password` password for HTTP BASIC authentication. Default is empty string (no authentication).
-// - `source` source that generated the event. Default is "Flux".
-//
-// The returned factory function accepts a `mapFn` parameter.
-// The `mapFn` must return an object with `node`, `metricType`, `resource`, `metricName`, `messageKey`, `description`,
-// `severity` and `additionalInfo` fields as defined in the `event` function arguments.
-//
-// ## Example
-//
-// ```
-//  import "contrib/bonitoo-io/servicenow"
-//  import "influxdata/influxdb/secrets"
-//  import "strings"
-//
-//  username = secrets.get(key: "SERVICENOW_USERNAME")
-//  password = secrets.get(key: "SERVICENOW_PASSWORD")
-//
-//  endpoint = servicenow.endpoint(
-//      url: "https://tenant.service-now.com/api/global/em/jsonv2",
-//      username: username,
-//      password: password
-//  )
-//
-//  from(bucket: "example-bucket")
-//    |> range(start: -1m)
-//    |> filter(fn: (r) => r._measurement == "cpu" and r._field == "usage_idle")
-//    |> last()
-//    |> endpoint(mapFn: (r) => ({
-//        node: r.host,
-//        metricType: strings.toUpper(v: r._measurement),
-//        resource: r.instance,
-//        metricName: r._field,
-//        severity:
-//            if r._value < 1.0 then "critical"
-//            else if r._value < 5.0 then "warning"
-//            else "info",
-//        additionalInfo: { "devId": r.dev_id }
-//      })
-//    )()
+// - url: (Required) ServiceNow web service URL.
+// - username: (Required) ServiceNow username to use for HTTP BASIC authentication.
+// - password: (Required) ServiceNow password to use for HTTP BASIC authentication.
```

Comment:
The "required" tags aren't needed. If a parameter is required, that metadata is included in the generated fluxdoc JSON and we can dynamically apply styles using that metadata.

```suggestion
// - url: ServiceNow web service URL.
// - username: ServiceNow username to use for HTTP BASIC authentication.
// - password: ServiceNow password to use for HTTP BASIC authentication.
```